fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/SchemaType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.database.utils.api;
+
+
+/**
+ * The flavor of database schema
+ *   PLAIN       - the schema we typically deploy to Derby or PostgreSQL
+ *   MULTITENANT - on Db2 supporting multiple tenants using partitioning and RBAC
+ *   DISTRIBUTED - for use with distributed technologies like Citus DB
+ *   SHARDED     - explicitly sharded using an injected shard_key column
```

Comment:
I think these would be better as javadoc on the actual values, but here is better than not having it :-)

Reply:
ok